pkg/events/derive/symbols_loaded.go

```diff
@@ -78,14 +81,18 @@ func (symbsLoadedGen *symbolsLoadedEventGenerator) deriveArgs(event trace.Event)
 	}
 
 	soSyms, err := symbsLoadedGen.soLoader.GetExportedSymbols(loadingObjectInfo)
+	// This error happens frequently in some environments, so we need to silence it to reduce spam.
+	// Either way, this is not a critical error so we don't return it.
 	if err != nil {
-		// TODO: Create a warning upon this error when logger is available
 		// TODO: rate limit frequent errors for overloaded envs
-		if symbsLoadedGen.isDebug {
-			return nil, err
+		_, ok := symbsLoadedGen.returnedErrors[err.Error()]
+		if !ok {
+			symbsLoadedGen.returnedErrors[err.Error()] = true
+			logger.Warn("symbols_loaded", "object loaded", loadingObjectInfo, "error", err.Error())
```

Comment:
That is a nice approach for repetitive errors. Could you split this change (in symbols_loaded) into another commit with proper git log ?